templates/2k11/admin/entries.tpl

```diff
@@ -42,11 +42,11 @@
             <button class="wrap_insimg" type="button" name="insJ" data-tarea="serendipity[body]">img</button>
             <button class="wrap_insmedia" type="button" name="insImage" data-tarea="serendipity[body]">{$CONST.MEDIA}</button>
             <button class="wrap_insurl" type="button" name="insURL" data-tarea="serendipity[body]">URL</button>
-            {serendipity_hookPlugin hook="backend_entry_toolbar_body" data=$entry_data.entry hookAll="true"}
```

Comment:
That looks like a major bug to me, doesn't this mean the event hook didn't work at all before?

Reply:
Right. I do not see `$entry_data` anywhere, it was not assigned.

Reply:
Odd that this didn't lead to reports, but good that it can get fixed. But I'd isolate this as a specific bugfix commit (and document in NEWS)

Reply:
Okay. But please not that it is also a PHP 8 compatibility fix, as the old code threw a warning there. This is one of the cases that is hard to separate.

Reply:
Separated into https://github.com/s9y/Serendipity/commit/e2682dbd79b6d37045226711a457ce544c27a395